### ISSUE TITLE
Compliance Report: What additional Reporting fields will be needed for Q1 of 2024? 

### ISSUE THREAD

**Issue by jaimiebutton** [bcgov/cas-reporting#14]
Background:
For the first year of OBPS we will need to include extra data fields in the Reporting app. 
This is because for Q1 of 2024 (Jan - Mar 31) Industry will still be paying carbon tax. Then the carbon tax exemption starts April 1 2024. The business area will need a way to calculate emissions/production data for those first 3 months. 

Acceptance Criteria: 
- [x] Work with ES & SMEs to determine what fields need to be included in Reporting so that CG staff can reconcile Q1 carbon tax payment issue.
- [x] Scope out how these fields could be used in future years if necessary (for example when an operation becomes regulated part-way through a year and need to report partial year data in a similar way)

DOD: 
- [x] Document final approved decision from Directors (MS/CF) on the approach

Goal of this ticket: 
- [x] To confirm the additional data fields required and get approval on the approach by Business Area.

**Comment by Dianadec** [comment]
Blocked by #50

**Comment by patriciarussellCAS** [comment]
DECISION: Feb 14 2024 (ES)
Here is what will be required in the compliance report for the 2024 reporting year only. This will help reconcile the first 3 months of 2024 where Industry operators still pay the carbon tax. This will not be required for future reporting years. 

1. Full Year Production data for 2024
2. April - Dec Production data for 2024 <this is the only additional data requested for 2024. 
3. Partial year compliance emissions  (will be calculated by info user provides). 

NOTE: These extra fields could also be used for Regulated Operations who become regulated part-way through a reporting year, so they can be used in future scenarios, not just year 1 (2024).

**Comment by suhafa** [comment]
Update: now that we know the answer, we need to ensure this is part of the reporting fields artifact that business area is working on currently

**Comment by patriciarussellCAS** [comment]
Closing this ticket as we believe this has been answered. Follow-up ticket to be created to capture next steps.
